tests/wasm/classes.rs

```diff
@@ -599,12 +599,12 @@ impl OverriddenInspectable {
     }
 
     #[wasm_bindgen(js_name = toJSON)]
-    pub fn to_json(&self) -> String {
+    pub fn js_to_json(&self) -> String {
```

Comment:
For `to_string`, the lint `inherent_to_string` suggests to implement the `Display` trait instead of adding an inherent method. That is unsupported with wasm_bindgen. Therefore, I changed the practically irrelevent Rust-side name to `js_to_string` to highlight that this is the JS-side `toString` method, not designed for Rust callers. Maybe `to_string_js` would be better? Then, I did the same for `to_json` for consistency.

Reply:
That's fine.